1-JavaScript/Week2/homework/ex1-giveCompliment.js

```diff
@@ -1,36 +1,29 @@
 'use strict';
-/* -----------------------------------------------------------------------------
-Full description at: https://github.com/HackYourFuture/Homework/tree/main/1-JavaScript/Week3#exercise-1-you-are-amazing
-
-1. Complete the function named `giveCompliment`as follows:
-
-   - It should take a single parameter: `name`.
-   - Its function body should include a variable that holds an array,
-     `compliments`, initialized with 10 strings. Each string should be a
-     compliment, like `"great"`, `"awesome"` and so on.
-   - It should randomly select a compliment from the array.
-   - It should return the string "You are `compliment`, `name`!", where
-     `compliment` is a randomly selected compliment and `name` is the name that
-     was passed as an argument to the function.
-
-2. Call the function three times, giving each function call the same argument:
-   your name.
-   Use `console.log` each time to display the return value of the
-   `giveCompliment` function to the console.
------------------------------------------------------------------------------*/
-function giveCompliment(/* TODO parameter(s) go here */) {
-  // TODO complete this function
+function giveCompliment(name) {
+const compliments = [
+  "wonderful",
+  "impressive",
+  "outstanding",
+  "marvelous",
+  "exceptional",
+  "brilliant",
+  "remarkable",
+  "splendid",
+  "extraordinary",
+ "magnificent"];
+const randomSelection= Math.floor(Math.random()*10);
+const compliment=compliments[randomSelection];
+return `You Are ${compliment}, ${name} !`
```

Comment:
Great work on this function 💪 I like how you stored the `randomSelection` in a different variable, makes it more readable and then it's easy to change!

Small tip: the code needs some spaces in between, for example the assignment and the multiplication to make it more readable. See the improvements below.

```suggestion
const randomSelection = Math.floor(Math.random() *10);
const compliment = compliments[randomSelection];
return `You Are ${compliment}, ${name} !`
```